samples/azure-quantum/dynamic-error-correction/DynamicErrorCorrection.ipynb

```diff
@@ -0,0 +1,369 @@
+        "    mutable corrections = 0;\n",
+        "    within {\n",
+        "        // Encode/Decode logical qubit.\n",
+        "        Encode(localRegister);\n",
+        "    }\n",
+        "    apply {\n",
+        "        LogicalX(localRegister);\n",
+        "        let iterations = 5;\n",
+        "        for _ in 1 .. iterations {\n",
+        "            // Apply unitary operations.\n",
+        "            ApplyLogicalOperation(localRegister);\n",
+        "\n",
+        "            // Perform error correction and increase the counter if a correction was made.\n",
+        "            let (parityA01, parityA12) = MeasureSyndrome(localRegister);\n",
+        "            let correctedError = CorrectError(localRegister, parityA01, parityA12);\n",
+        "            if (correctedError) {\n",
+        "                set corrections += 1;\n",
+        "            }\n",
+        "        }\n",
+        "    }\n",
+        "\n",
+        "    // Measure the first qubit in register and return value.\n",
+        "    let result = MResetZ(localRegister[0]);\n",
+        "    ResetAll(localRegister);\n",
+        "    return (result, corrections);\n",
+        "}"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 7. Submit the DynamicBitFlipCode operation\n",
+        "For 50 shots, as configured below, the following will use about 11.31 units (HQCs or eHQCs).<br/>\n",
+        "If the job is not completed after the timeout (300 seconds), you can query the status followed by querying the output.<br/>\n",
+        "You can also check the status under Operations > Job management.\n",
+        "\n",
+        "**Understanding the results**<br/>\n",
+        "The histogram reports the ratio of the various outputs.</br>\n",
+        "For instance, if you get:</br>\n",
+        "- `Result: (1,0) Frequency: 0.92` means that in 92% of the total number of shots the operation returned a qubit with the value One and that no error correction was necessary.</br>\n",
+        "- `Result: (0,2) Frequency: 0.02` means that in 2% of the total number of shots the operation returned a qubit with the value Zero and that 2 error corrections were performed. The error correction failed as the value flipped.</br>\n",
+        "- `Result: (0,0) Frequency: 0` means that in 2% of the total number of shots the operation returned a qubit with the value One and that 2 error corrections were performed. The error correction failed as the value flipped.</br>\n",
+        "- `Result: (1,2) Frequency: 0.02` means that in 2% of the total number of shots the operation returned a qubit with the value One and that 2 error corrections were performed."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "%azure.target quantinuum.sim.h1-1e-preview\n",
```

Comment:
should target be 1e or 1e-preview? @cesarzc @masenol

Reply:
We shouldn't target `-preview` targets as they are not going to be productionalized, but the QIR flow will be exposed on existing targets at production without any '-preview' suffix. When it comes to which target to point to, last decision is to use a simulator target instead of apival (syntax checker) or qpu targets.

Quantinuum has multiple simulators i.e. H1-1e, He-2e. @shenyingjun Could you check with Fabrice which production simulator target to use here?